### PR TITLE
Add DPI marketing page

### PR DESCRIPTION
**Pull request by bjss-colin**

### CODE REVIEW

**Comment by bjss-colin** [comment]
Converting to draft as this depends on the expander changes in #1481

**Review by dssherlock — APPROVED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/DataProcessingInformation.cshtml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+@model NHSD.GPIT.BuyingCatalogue.WebApp.Areas.Solutions.Models.DataProcessingInformationModel
+
+<article>
+    <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+            <partial name="_solutionSummary" model="Model"/>
+
```
Comment (dssherlock) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/DataProcessingInformation.cshtml:7]: Are both pieces of information required for this to become visible? I.e. if someone has added the General Information, should this still be hidden until the Officer is added too?

Comment (ghost) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/DataProcessingInformation.cshtml:7]: Yeah that's right. Details & DPO are mandatory while Sub-processors is optional

Comment (dssherlock) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/DataProcessingInformation.cshtml:7]: Ok, and presumably the marketing pages only show published solutions. Not ones in progress.

Comment (ghost) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/DataProcessingInformation.cshtml:7]: That's correct. Only Published solutions will have all pages displayed

**Pull request merged**